cLvgl/Dockerfile.sdk

```diff
@@ -0,0 +1,43 @@
+# ARGUMENTS --------------------------------------------------------------------
+ARG CROSS_SDK_BASE_TAG=3.3.1
+
+##
+# Board architecture
+##
+ARG IMAGE_ARCH=
+
+##
+# Application root directory inside the container
+##
+ARG APP_ROOT=
+
+
+# BUILD ------------------------------------------------------------------------
+FROM commontorizon/debian-cross-toolchain-${IMAGE_ARCH}:${CROSS_SDK_BASE_TAG}
```

Comment:
We do not use `commontorizon/` images anymore, so here should be `torizon/debian-croos-toolchain-${IMAGE_ARCH}:${CROSS_SDK_BASE_TAG}`